apps/site/pages/en/about/index.mdx

```diff
@@ -44,6 +44,14 @@ server.listen(port, hostname, () => {
 });
 ```
 
+```cjs
```

Comment:
But that is intentional. Sequential codeboxes are supposed to be merged together afaik.

Reply:
take a look to the issue. In doc it's supposed to be separated

Reply:
Then let's follow it the way it is supposed to be.